contracts/AMM/ThalesAMM.sol

```diff
@@ -125,7 +128,7 @@ contract ThalesAMM is ProxyOwned, ProxyPausable, ProxyReentrancyGuard, Initializ
             if (basePrice <= MIN_SUPPORTED_PRICE || basePrice >= MAX_SUPPORTED_PRICE) {
                 return 0;
             }
-            uint sell_max_price = basePrice.mul(ONE.sub(max_spread)).div(ONE);
+            uint sell_max_price = basePrice.mul(ONE.sub(max_spread.sub(safeBoxImpact))).div(ONE);
```

Comment:
safeBoxImpact is an amount, not a percentage? so because of that is sub(traction)

Reply:
its a percentage, but at that place we calculate the percentage to add to base price